jarviscli/plugins/movie.py

```diff
@@ -223,11 +223,16 @@ def get_movie_info(jarvis, data):
     """
     global movie_attributes
```

Comment:
Is this line required?

Reply:
I don't think it is required, because the variable is not used in any other functions. I left it unchanged because it was global before I started working on this issue. I am going to change it and do a commit.